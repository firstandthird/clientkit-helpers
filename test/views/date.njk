{{ '' | date }}
{{ '2017-11-5' | date('dddd, MMMM Do YYYY') }}
{{ '11-05-2017' | date('YYYY/MM/DD', 'MM-DD-YYYY') }}
