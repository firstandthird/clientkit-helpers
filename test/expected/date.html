a few seconds ago
Sunday, November 5th 2017
2017/11/05
